srcpkgs/greetd/INSTALL.msg

```diff
@@ -0,0 +1 @@
+In order for agreety to work correctly, disable the agetty service for the tty
```

Comment:
not sure if this is relevant on the latest versions

Reply:
Or maybe should be added to `README.voidlinux` instead of INSTALL.msg